styles.css

```diff
@@ -0,0 +1,38 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 718px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+}
+
+.introduction {
+  display: flex;
+  flex-direction: column;
+  height: 604px;
+}
+
+.header {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.intro {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
```

Comment:
- [ ] Please take note that the font family of these elements is `Poppins`, you may follow [this short tutorial](https://www.w3schools.com/css/css_font_google.asp). 💁‍♂️